packages/react/alert-dialog/src/AlertDialog.tsx

```diff
@@ -80,10 +80,9 @@ const AlertDialogContent = React.forwardRef<AlertDialogContentElement, AlertDial
 
     return (
       <DialogPrimitive.LabelWarningProvider
-        value={React.useMemo(
-          () => ({ contentName: CONTENT_NAME, titleName: TITLE_NAME, docsSlug: 'alert-dialog' }),
-          []
-        )}
+        contentName={CONTENT_NAME}
+        titleName={TITLE_NAME}
+        docsSlug="alert-dialog"
```

Comment:
I'm confused how this one gets a change when I don't see any associated change to `LabelWarningProvider`.
What am I missing here?

Reply:
The `LabelWarningProvider` change is here https://github.com/radix-ui/primitives/pull/852/files/3e240a58c85a0cd28a02a1379388a83a48b95038#diff-18be67c22def126bf675c06beb358c92a4a8c6d1a071adf162fcd5ac9cac18bdR422

It's using  `DialogPrimitive.LabelWarningProvider`.